packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -145,7 +142,7 @@ class GraphPack {
   std::vector<std::vector<GO> > element2node;
 
   // NOTE: This is hardwired for 1D bar elements
-  typedef Kokkos::View<LO*[2], Kokkos::LayoutLeft, typename Node::device_type > k_element2node_type;
+  typedef Kokkos::View<GO*[2], Kokkos::LayoutLeft, typename Node::device_type > k_element2node_type;
```

Comment:
Prefer C++11 style `using new_name_type = OldName;`

Reply:
IDK why this is even typedef'd, it's not used anywhere else.  Gone.